submissions/Nikita_Lemesiev/css_and_html_practice/index.html

```diff
@@ -0,0 +1,112 @@
+                            <div tabindex="0" class="item">
+                                <img src="documents.png" alt="Documents">
+                                Documents
+                            </div> 
+                            <div tabindex="0" class="item">
+                                <img src="files.png" alt="Apps">
+                                Files
+                            </div> 
+                            <div tabindex="0" class="item">
+                                <img src="mail.png" alt="Mail">
+                                Mail
+                            </div> 
+                            <div tabindex="0" class="item">
+                                <img src="maps.png" alt="Maps">
+                                Maps
+                            </div> 
+                            <div tabindex="0" class="item">
+                                <img src="mars.png" alt="Mars">
+                                Mars
+                            </div>
+                            <input tabindex="0" id="more_checker" type="checkbox" name="more">
+                            <label  for="more_checker" id="more_icons_label">More</label>
+                                <div id="more_icons">
+                                    <div tabindex="0" class="item">
+                                        <img src="news.png" alt="News">
+                                        News
+                                    </div>
+                                    <div tabindex="0" class="item">
+                                        <img src="photos.png" alt="Photos">
+                                        Photos
+                                    </div>
+                                    <div tabindex="0" class="item">
+                                        <img src="promo.png" alt="Promo">
+                                        Promo
+                                    </div>
+                                    <div tabindex="0" class="item">
+                                        <img src="search.png" alt="Search">
+                                        Search
+                                    </div>
+                                    <div tabindex="0" class="item">
+                                        <img src="tanslation.png" alt="Translation">
+                                        Translation
+                                    </div>
+                                    <div tabindex="0" class="item">
+                                        <img src="videos.png" alt="Apps">
+                                        Videos
+                                    </div>
+                                </div>
+                        </div>
+                </li>
+                <li><img id="notification" src="bell.png"></li>
+                <li><img id="avatar" src="avatar.jpg"></li>
+            </ul>
+            
+        </nav>
+    </header>
+</body>
+</html>
```

Comment:
Always add an empty line to the end of the file. You can set your code editor to do so when you save a file. It helps some programs determine where the file ends, among other reasons.